sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/UniqueIndexTest.java

```diff
@@ -222,7 +222,7 @@ public void before_UniqueIndexTest() {
         client = new CosmosClientBuilder()
             .endpoint(TestConfigurations.HOST)
             .key(TestConfigurations.MASTER_KEY)
-            .connectionPolicy(ConnectionPolicy.getDefaultPolicy())
+            .directMode(DirectConnectionConfig.getDefaultConfig())
```

Comment:
Do these tests are always expected to run in Direct mode?

Reply:
Yes, I have not changed the connection mode of the tests - they will run as it is as they were running earlier. 
Since earlier also, default was DIRECT mode.